hooks/post_gen_project.py

```
@@ -36,28 +36,36 @@ def init_git():
         git.wait()
 
 def init_proto():
-    # Order matters: download → generate (needs buf plugins) → mock (needs generated
-    # interfaces) → tidy (needs all imports including generated mocks to resolve)
+    # Order matters: download → generate (needs buf plugins) → tidy -e (reconcile
+    # indirect deps; -e ignores missing mock packages) → mock (needs generated
+    # interfaces) → tidy (final, needs all imports including mocks to resolve)
     print("Starting proto initialization...")
-    print("Step 1/4: Fetching Go modules (this might take a few minutes)...")
+    print("Step 1/5: Fetching Go modules (this might take a few minutes)...")
     code = Popen(["go", "mod", "download", "all"], cwd=PROJECT_DIRECTORY).wait()
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)